### PR TITLE
Handle host header

### PR DESCRIPTION
**Pull request by freeformz**
Do it case insensitively.

### CODE REVIEW

**Comment by freeformz** [comment]
Since according to the docs Request.Host is used instead of a host header.

File: main.go [hunk 1]
```diff
@@ -183,7 +183,12 @@ func visit(url *url.URL) {
 		log.Fatalf("unable to create request: %v", err)
 	}
 	for _, h := range httpHeaders {
-		req.Header.Add(headerKeyValue(h))
+		k, v := headerKeyValue(h)
+		if strings.EqualFold(k, "host") {
```
Comment (theckman) [inline main.go]: I can't think of a case where you would need to use `EqualFold()` for an HTTP header field-name. They should only be ASCII-compatible bytes.

Comment (theckman) [inline main.go]: Do we want `host` or `Host`? We should probably use this function to ensure they are following the same format rules:

* https://golang.org/pkg/net/http/#CanonicalHeaderKey

Comment (theckman) [inline main.go]: Sorry, I take back this statement. I was unaware that `strings.EqualFold()` also included changes in the capitalization of letters.

:+1:

Comment (theckman) [inline main.go]: Answered my own question: https://github.com/davecheney/httpstat/pull/71/files#r80565114

Sorry for the noise.

**Comment by davecheney** [comment]
Thanks,

**Comment by tcnksm** [comment]
I missed that. Thanks !

**Pull request merged**